scripts/build_lib_power.py

```diff
@@ -10,7 +10,7 @@
 import os.path
 
 PWR_NAMES = [
-    "VCC", "VDD", "AVCC", "1v2", "1v8", "3v3", "5v",  "12v", "VBATT", "VSHORE",
+    "VCC", "VDD", "AVCC", "1v2", "1v8", "3v3", "5v",  "12v", "VBATT", "VSHORE", "3v3_B", "5v_B"
```

Comment:
Not convinced by these power symbols for general inclusion in agg-kicad. If you just need a second rail for a specific thing consider making a project-specific symbol library for it, or using a named net.